code/modules/mob/living/simple_animal/abnormality/he/jangsan.dm

```diff
@@ -93,7 +92,7 @@
 	var/chance_modifier = 1
 	StatCheck(user)
 	for(var/z in 0 to strong_counter)
-		chance_modifier *= 0.9
+		chance_modifier *= 0.85
```

Comment:
I feel like this is gonna nullify most of the change the work rates are gonna have

Reply:
It scales a lot better this way.